src/components/SceneConfigMenu.js

```diff
@@ -222,14 +222,18 @@ class ConfigModal extends Component {
 
   // Toggles whether the editor is showing
   viewToggle = () => {
-    let style = this.props.scene.settings.viewOnly ? btnStyle.off : btnStyle.on;
+    let viewOnly = this.props.scene.settings.viewOnly
```

Comment:
This is not technically wrong, but it adds unnecessary complexity to the code. It is best to only use a variable to store a prop when you're using it multiple times or in long strings. In addition, props stored as a variable should always be a const as you can't change them.